packages/analytics/__tests__/analytics.test.ts

```diff
@@ -843,10 +877,26 @@ describe('Analytics', function () {
       expect(initiateOnDeviceConversionMeasurementWithEmailAddress).toBeDefined();
     });
 
+    it('`initiateOnDeviceConversionMeasurementWithHashedEmailAddress` function is properly exposed to end user', function () {
+      expect(initiateOnDeviceConversionMeasurementWithHashedEmailAddress).toBeDefined();
+    });
+
+    it('`initiateOnDeviceConversionMeasurementWithHashedPhoneNumber` should throw if the value is in E.164 format', function () {
+      expect(() =>
+        initiateOnDeviceConversionMeasurementWithHashedPhoneNumber(getAnalytics(), '+1234567890'),
+      ).toThrowError(
+        "firebase.analytics().initiateOnDeviceConversionMeasurementWithHashedPhoneNumber(*) 'hashedPhoneNumber' expected a sha256-hashed value of a phone number in E.164 format.",
+      );
+    });
+
```

Comment:
@matinzd - Could you also write modular test for this one as well, please?

```js
it('throws if not a string', function () {
      expect(() =>
        // @ts-ignore
        firebase.analytics().initiateOnDeviceConversionMeasurementWithHashedEmailAddress(true),
      ).toThrowError(
        "firebase.analytics().initiateOnDeviceConversionMeasurementWithHashedEmailAddress(*) 'hashedEmailAddress' expected a string value.",
      );
    });
```

That should be it I think 😄

Reply:
Done :D